documentation/modules/post/windows/manage/persistence_service.rb

```diff
@@ -0,0 +1,100 @@
+## Overview
+This Module will generate and upload an executable to a remote host and make it persistent service.
```

Comment:
Make it *a* persistent.
Or, since this has two 'and's: remote host. Next it will make the executable a persistent...